tests/Unit/Domain/CoordinateMaps/Test_EquatorialCompression.cpp

```diff
@@ -46,36 +46,44 @@ void test_radius() {
   const double aspect_ratio = aspect_ratio_dis(gen);
   CAPTURE(aspect_ratio);
   const std::array<double, 3> input_point{{x, y, z}};
-  const CoordinateMaps::EquatorialCompression angular_compression_map(
-      aspect_ratio);
-  const auto result_point = angular_compression_map(input_point);
-  const auto& result_x = result_point[0];
-  const auto& result_y = result_point[1];
-  const auto& result_z = result_point[2];
 
-  // The EquatorialCompression map should preserve radial lengths.
-  CHECK(magnitude(result_point) == approx(magnitude(input_point)));
+  for (const auto& index_polar_axis : {0ul, 1ul, 2ul}) {
```

Comment:
(optional) You can use `1_st`, `2_st` etc for size_t literals, as defined in Utilities/Literals.hpp.